ontology/spdx-ontology.owl.xml

```diff
@@ -0,0 +1,3001 @@
+<?xml version="1.0"?>
+
+
+<!DOCTYPE rdf:RDF [
+    <!ENTITY foaf "http://xmlns.com/foaf/0.1/" >
+    <!ENTITY owl "http://www.w3.org/2002/07/owl#" >
+    <!ENTITY xsd "http://www.w3.org/2001/XMLSchema#" >
+    <!ENTITY rdfs "http://www.w3.org/2000/01/rdf-schema#" >
+    <!ENTITY rdf "http://www.w3.org/1999/02/22-rdf-syntax-ns#" >
+    <!ENTITY ns "http://www.w3.org/2003/06/sw-vocab-status/ns#" >
+]>
+
+
+<rdf:RDF xmlns="http://www.semanticweb.org/gary/ontologies/2014/6/untitled-ontology-6#"
+     xml:base="http://www.semanticweb.org/gary/ontologies/2014/6/untitled-ontology-6"
+     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+     xmlns:ns="http://www.w3.org/2003/06/sw-vocab-status/ns#"
+     xmlns:foaf="http://xmlns.com/foaf/0.1/"
+     xmlns:owl="http://www.w3.org/2002/07/owl#"
+     xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
+     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
+    <owl:Ontology rdf:about="http://www.semanticweb.org/spdx/ontologies/2014/6/SPDX-2.1">
+        <rdfs:label>SPDX 2.1</rdfs:label>
+        <owl:versionInfo>2.1</owl:versionInfo>
+        <rdfs:comment>This specification describes the SPDX® language, defined as a dictionary of named properties and classes using W3C&apos;s RDF Technology.
+
+SPDX® is a designed to allow the exchange of data about software packages. This information includes general information about the package, licensing information about the package as a whole, a manifest of files contained in the package and licensing information related to the contained files.
+
+Known issues:
+ - File fileType cardinality should be 0 or more instead of required one.</rdfs:comment>
+    </owl:Ontology>
+    
+
+
+    <!-- 
+    ///////////////////////////////////////////////////////////////////////////////////////
+    //
+    // Annotation properties
+    //
+    ///////////////////////////////////////////////////////////////////////////////////////
+     -->
+
+    
+
+
+    <!-- http://www.w3.org/2002/07/owl#deprecatedClass -->
+
+    <owl:AnnotationProperty rdf:about="&owl;deprecatedClass"/>
+    
+
+
+    <!-- http://www.w3.org/2002/07/owl#deprecatedProperty -->
+
+    <owl:AnnotationProperty rdf:about="&owl;deprecatedProperty"/>
+    
+
+
+    <!-- http://www.w3.org/2003/06/sw-vocab-status/ns#term_status -->
+
+    <owl:AnnotationProperty rdf:about="&ns;term_status">
+        <ns:term_status>stable</ns:term_status>
+        <rdfs:comment>The AnyLicenseInfo class includes all resources that represent licensing information.</rdfs:comment>
+        <rdfs:isDefinedBy>http://spdx.org/rdf/terms#</rdfs:isDefinedBy>
+    </owl:AnnotationProperty>
+    
+
+
+    <!-- 
+    ///////////////////////////////////////////////////////////////////////////////////////
+    //
+    // Object Properties
+    //
+    ///////////////////////////////////////////////////////////////////////////////////////
+     -->
+
+    
+
+
+    <!-- http://spdx.org/rdf/terms#agent -->
```

Comment:
Tool generated

Reply:
> Tool generated

If we land the OWL in this repo, we'll want to agree on how it will be maintained.  If the idea is to conform to the output of a particular tool, it would be good to document that tool so other contributors can use it to generate matching OWL.  Otherwise I think we'll end up with unnecessary churn as some contributors edit by hand (or with different tools), and then your tool clobbers those changes to return to its preferred output format.